example/frontend/index.js

```diff
@@ -21,5 +21,6 @@ const viewer = new FrameContainer(container, {
   relayPageURL: process.env.CONFIG_RELAY_PAGE_URL,
   useOpaqueOrigin: Boolean(process.env.CONFIG_USE_OPAQUE_ORIGIN),
   imageProxyURL: process.env.CONFIG_IMAGE_PROXY_URL,
+  loadTimeout: 3000,
```

Comment:
is this something that should discussed and standardized across all email clients? Sounds like it should be as users should expect a consistent experience across all ISPs?

Reply:
Potentially. I feel the timeout may depend on an email client's underlying infrastructure, but we should try to standardize as much of this as possible. Opened #38 to track this.